src/Tribe/Attendee_Registration/Shortcode.php

```diff
@@ -56,44 +56,7 @@ public function maybe_enqueue_scripts() {
 	public function render() {
 		ob_start();
 
-		// things we need added, arrays so we can filter more if necessary
-		$additional_allowed = [
-			// forms
-			'form' => [
-				'class'  => [],
-				'id'     => [],
-				'name'   => [],
-				'value'  => [],
-				'type'   => [],
-				'action' => [],
-				'method' => [],
-			],
-			// form fields - input
-			'input' => [
-				'class'   => [],
-				'id'      => [],
-				'name'    => [],
-				'value'   => [],
-				'type'    => [],
-				'checked' => [],
-			],
-			// select
-			'select' => [
-				'class'  => [],
-				'id'     => [],
-				'name'   => [],
-				'value'  => [],
-				'type'   => [],
-			],
-			// select options
-			'option' => [
-				'selected' => [],
-			],
-		];
-
-		$allowed_tags = array_merge( wp_kses_allowed_html( 'post' ), $additional_allowed );
-
-		echo wp_kses( tribe( 'tickets.attendee_registration.view' )->display_attendee_registration_shortcode(), $allowed_tags );
+		echo tribe( 'tickets.attendee_registration.view' )->display_attendee_registration_shortcode();
```

Comment:
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'tribe'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: a2ce04933cea0d91b09bcaa6467f72eb</sub>